clients/fides-js/src/lib/overlay.module.css

```diff
@@ -2,61 +2,49 @@
 * Default CSS styles for the banner
 */
 :root {
-  --fides-consent-overlay-font-family: inherit;
-  --fides-consent-overlay-font-size: 16px;
-  --fides-consent-overlay-font-size-title: 18px;
-  --fides-consent-overlay-background: #fff;
-  --fides-consent-overlay-text-color: #24292f;
-  --fides-consent-overlay-padding: 0.75em 2em 1em;
-
-  --fides-consent-overlay-button-background: #eee;
-  --fides-consent-overlay-button-text-color: #24292f;
-  --fides-consent-overlay-button-hover-background: #ccc;
-  --fides-consent-overlay-button-border-radius: 4px;
-  --fides-consent-overlay-button-padding: 0.5em 1em;
-
-  --fides-consent-overlay-button-primary-background: #464b83;
-  --fides-consent-overlay-button-primary-text-color: #fff;
-  --fides-consent-overlay-button-primary-hover-background: #3f4375;
-
-  --fides-consent-overlay-button-secondary-background: var(
-    --fides-consent-overlay-button-background
-  );
-  --fides-consent-overlay-button-secondary-text-color: var(
-    --fides-consent-overlay-button-text-color
-  );
-  --fides-consent-overlay-button-secondary-hover-background: var(
-    --fides-consent-overlay-button-hover-background
-  );
-
-  --fides-consent-overlay-button-tertiary-background: var(
-    --fides-consent-overlay-button-background
-  );
-  --fides-consent-overlay-button-tertiary-text-color: var(
-    --fides-consent-overlay-button-text-color
-  );
-  --fides-consent-overlay-button-tertiary-hover-background: var(
-    --fides-consent-overlay-button-hover-background
-  );
-
-  /* In progress(fides#3258): start to transition to updated design colors */
+  /* Colors */
   --fides-consent-overlay-primary-color: #8243f2;
   --fides-consent-overlay-background-color: #f7fafc;
+  /* Buttons */
+  --fides-consent-overlay-primary-button-background-color: #824ef2;
```

Comment:
Quick comment: if you use `var(--fides-consent-overlay-primary-color)` here instead, you can reuse that one branded color as much as possible, so that we don't need to override the button colors _unless_ we want them to be different.

That'd also fix what I think is a typo here where the button color should probably be #8243f2?

You can repeat this feedback for the other global vars too: as much as possible, let's reuse generic vars like "primary color", "font color", etc. and reduce the surface for forgetting to keep certain values to match each other.